src/Native/System.Security.Cryptography.Native/pal_ssl.cpp

```diff
@@ -496,13 +440,6 @@ static ExchangeAlgorithmType MapExchangeAlgorithmType(const SSL_CIPHER* cipher)
 
         case SSL_KeyExchangeAlgorithm::SSL_kSRP:
             return ExchangeAlgorithmType::SSL_kSRP;
-#else
-        case SSL_KeyExchangeAlgorithm::SSL_kECDH:
-            return ExchangeAlgorithmType::SSL_ECDH;
-
-        case SSL_KeyExchangeAlgorithm::SSL_kECDHE:
-            return ExchangeAlgorithmType::SSL_ECDSA;
```

Comment:
@vijaykota @CIPop When deleting this I noticed that we have what looks like some crossover between ECDH and ECDSA (it also exists in the non-deleted code, but I can't flag that in this PR).  Are we matching a Windows behavior here, or do we have a data mapping bug in Unix?

I wouldn't want to fix it in this PR if there's a bug; but it seems a reasonable place to have the code in question linked to.

(@ericeil FYI)